client/views/components/generic/dropdown/dropdown.scss

```diff
@@ -26,7 +26,7 @@
         transform: translate(-50%);
         padding: 10px 0 20px;
         margin: 0;
-        width: 100%;
+        //width: 100%;
```

Comment:
try to open the bucket on mobile version and try to move a card. it used the dropdown and i am not sure if this will break it or not